frontend/pages/settings.tsx

```diff
@@ -68,6 +79,16 @@ const Settings = ({ userInfo, authenticated }: SettingsProps) => {
       icon: 'bookmark',
       content: <FavoritesTab key="subscription" keyword="subscription" />,
     },
+    {
+      name: 'submissions',
+      label: 'Submissions',
+      content: <SubmissionsPage initialSubmissions={submissions} />,
+    },
+    {
+      name: 'applications',
+      label: 'Applications',
+      content: <ApplicationsPage whartonapplications={whartonapplications} />,
```

Comment:
Now that the page is more visible in the profile, would be nice to clarify that it's for Wharton Cycle applications specifically (maybe change it to all eventually but not needed now).